.circleci/config.yml

```diff
@@ -133,6 +133,25 @@ jobs:
           path: cypress/videos
       - store_artifacts:
           path: cypress/screenshots
+  test-a11y:
+    docker:
+      - image: circleci/node:10.15.3-browsers
+    working_directory: ~/repo
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - v7-dependencies-{{ checksum "yarn.lock" }}
+
+      # PR's from forks cannot use the dependency cache for performance reasons
+      - run:
+          name: 'Forked PR dependency install'
+          command: yarn
+
+      - run:
+          name: Accessibility Audit
+          command: node browser-test-harness.js yarn test:accessibility 
```

Comment:
can we get the json output added as an artifact?

Reply:
done!